src/core.rs

```diff
@@ -258,79 +227,92 @@ impl Internal {
             modifiers: Default::default(),
             resized: false,
         };
-        while running {
+
+        event_loop.run_return(|event, _, flow| {
+            // for compatibility with previous versions, which used an infinite run loop
+            *flow = ControlFlow::Poll;

```

Comment:
Polling seems like the "safe default," (e.g. the user is doing something time based) but I'm open if we want to expose a `Poll` vs `Wait` argument somewhere/somehow

Reply:
Done